sonar-cxx-plugin/pom.xml

```diff
@@ -100,6 +100,16 @@
       <version>2.6</version>
       <type>jar</type>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
It is widely used where ever I have seen JSON being processed with Java. No other than that.